### PR TITLE
Fix/utility namespace

### PR DESCRIPTION
**Pull request by RV-Adrian**

### CODE REVIEW

**Comment by RV-Adrian** [comment]
@alex-deaconu @ezequielc @fjvallarino 
Dependant PR for fix/utility-namespace

File: bower.json [hunk 1]
```diff
@@ -49,6 +49,9 @@
     "angular": "~1.5.0",
     "angular-mocks": "^1.5.5",
     "angular-sanitize": "~1.5.0",
-    "angular-bootstrap": "~0.13.0"
+    "angular-bootstrap": "~0.13.0",
+    "rv-common-style": "v1.4.0",
+    "common-style": "v1.4.0",
```
Comment (alex-deaconu) [inline bower.json]: @RV-Adrian regarding the unnecessary resolution I mentioned in the other PR (https://github.com/Rise-Vision/common-app-components/pull/66), this is why you had to add this in. If you remove that resolution, you can remove this one too. Thanks.

**Comment by alex-deaconu** [comment]
@RV-Adrian see comment re: bower resolutions. Otherwise, looks good. Thanks!

**Comment by alex-deaconu** [comment]
@RV-Adrian looks good. Don't forget to increment the CH version and update the bower.json file after the other 2 PRs are merged. Thanks!

**Pull request merged**